infra/aws.conf

```diff
@@ -172,7 +172,7 @@ plugins {
         enabled = true
         use-default-credential-provider = true
         default-bucket = "nexus-delta-production"
-        prefix = ""
+        prefix = "/sbo/data/project/"
```

Comment:
Won't this break something? @imsdu @olivergrabinski 
It's what we have on prem but the folders do not yet exist in the `nexus-delta-production` bucket

Reply:
It is ok, I think